RPIPs/RPIP-29.md

```diff
@@ -24,99 +24,99 @@ This file SHALL be updated monthly in tandem with the GMC treasury update DAO po
+| [RA032304](https://dao.rocketpool.net/t/july-2023-gmc-call-for-retrospective-award-applications-deadline-is-july-15th/1938/4)        | IMC Work                      |   August 1, 2023 | Valdorff              | [valdorff.eth](https://etherscan.io/address/0xf280EF6d278a485f6fEb9d3242fd65d09633ec33)                                                                                                                         |            221.74 RPL | [transaction](https://etherscan.io/tx/0xe85854282e840efdd398d549e1c819a40453ef57f258ea8ca8c2a2dfcc68f0cd)                                                                                                                |
+| [RA032306](https://dao.rocketpool.net/t/july-2023-gmc-call-for-retrospective-award-applications-deadline-is-july-15th/1938/7)        | RocketWatch Part 2            |   August 1, 2023 | invis                 | [vault.0xinvis.eth](https://etherscan.io/address/0x75Cf8e1F8F4fbF4C7BB216E450BCff5F51Ab3E5A)                                                                                                                    |          1,739.14 RPL | [transaction](https://etherscan.io/tx/0xe85854282e840efdd398d549e1c819a40453ef57f258ea8ca8c2a2dfcc68f0cd)                                                                                                                |
+| [RA032307](https://dao.rocketpool.net/t/july-2023-gmc-call-for-retrospective-award-applications-deadline-is-july-15th/1938/7)        | The Weekly Orbit              |   August 1, 2023 | Pat, Waq              | [address](https://etherscan.io/address/0xc505a2dA2d34D76aE9c765A6AC926990E31b7cc5) [address](https://etherscan.io/address/0xDCC3A6Fea47ea451D5Cba2de39aAc17537421a7D)                                           |            140.87 RPL | [transaction](https://etherscan.io/tx/0xe85854282e840efdd398d549e1c819a40453ef57f258ea8ca8c2a2dfcc68f0cd)                                                                                                                |
+| [RA022312](https://dao.rocketpool.net/t/april-2023-gmc-call-for-retrospective-award-applications-deadline-is-april-15th/1639/16)     | ETH Denver Volunteers         |    April 3, 2023 | ETH Denver Volunteers | [list of address'](https://poap.gallery/event/111015) and [neurounique.eth](https://etherscan.io/address/0x72eC7bd6c4EeC85eFFDd2b32efa836F2FfEF2F69)                                                            |            627.44 RPL | [transaction](https://etherscan.io/tx/0xecf4cea2e4d5de4aa3176419d3b9ff9c4429dd12e1c5b2fcce9b48d3a17057b7)                                                                                                                |
```

Comment:
"list of addresses"

Reply:
fixed - this was pre-existing but I missed it on review so thanks for pointing it out